docs/work-flow.md

```diff
@@ -0,0 +1,176 @@
+# DC/OS Development Workflow
+
+
+        "... there is absolutely nothing "obvious" about source-control systems, or workflows before you already know them"
+
+Mesosphere DC/OS has two primary repositories.
+
+1. Open Source DC/OS (https://github.com/dcos/dcos)
+2. Closed Source Mesosphere Enterprise DC/OS (https://github.com/mesosphere/dcos-enterprise)
```

Comment:
The name changed at some point from Enterprise DC/OS to DC/OS Enterprise. Please change that throughout this document.